data/contributors.js

```diff
@@ -1687,6 +1688,15 @@ contributors.unshift(
     github: "https://github.com/Debajyoti14",
     gender: "Male",
   },
+ {

```

Comment:
This is still inside the contributors.unshift .

Reply:
Done ✅